File: Shows/ShowPHP/Sandwich.php
<?php

namespace TheRayCode\Builder;

class Sandwich
{
    private $bread;
    private $meat;
    private $cheese;
    private $veggies;

    function __construct($bread, $meat, $cheese, $veggies) {
        $this->bread = $bread;
        $this->meat = $meat;
        $this->cheese = $cheese;
        $this->veggies = $veggies;
    }

    function __toString() {
        return $this->bread . ", " . $this->meat . ", " . $this->cheese . ", " . $this->veggies;
    }
}

File: Shows/ShowCsharp/Show/Show/Sandwich.cs
using System;
using System.Collections.Generic;

namespace Show
{
    public class Sandwich
    {
        public string Bread { get; set; }
        public string Meat { get; set; }
        public string Cheese { get; set; }
        public List<string> Vegetables { get; set; }

        public void Display()
        {
            Console.WriteLine("Enjoy your delicious sandwich:");
            Console.WriteLine($"Bread: {Bread}");
            Console.WriteLine($"Meat: {Meat}");
            Console.WriteLine($"Cheese: {Cheese}");
            Console.WriteLine("Vegetables:");
            foreach (var vegetable in Vegetables)
            {
                Console.WriteLine($"- {vegetable}");
            }
        }
    }
}